cmd/pipeline/deploy.go

```diff
@@ -310,3 +312,9 @@ func getCurrentNamespace(ctx context.Context) string {
 	}
 	return os.Getenv("OKTETO_NAMESPACE")
 }
+
+func getPipelineURL(namespace string) string {
+	oktetoURL := okteto.GetURL()
+	previewURL := fmt.Sprintf("%s/#/spaces/%s", oktetoURL, namespace)
```

Comment:
```suggestion
	pipelineURL := fmt.Sprintf("%s/#/spaces/%s", oktetoURL, namespace)
```

Reply:
I think there is an url for the pipeline. that way, pipeline logs will be opened